harvey_plan9.go

```diff
@@ -0,0 +1,71 @@
+// +build plan9
```

Comment:
this build tag is unnecessary and ineffective.

build tags need a fully empty new line after them to be effective.

however, naming the file _plan9.go already added this build tag, so it's unnecessary.